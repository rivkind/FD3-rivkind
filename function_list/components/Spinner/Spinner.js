import React from 'react';

import './Spinner.css';

class Spinner extends React.PureComponent {

  render() {

    return (
      <div className="backdrop">
        <div className="loader"> 
          <div className="dot"></div>
          <div className="dot"></div>
          <div className="dot"></div>
          <div className="dot"></div>
          <div className="dot"></div>
         </div>
      </div>
    );
  }

}

export default Spinner;