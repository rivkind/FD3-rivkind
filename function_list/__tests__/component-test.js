"use strict";

import React from 'react';
import renderer from 'react-test-renderer';

import Spinner from '../components/Spinner/Spinner';

test('работа Spinner', () => {

  const component = renderer.create(
    <Spinner />
  );

  let componentTree=component.toJSON();
  expect(componentTree).toMatchSnapshot();
    
});
