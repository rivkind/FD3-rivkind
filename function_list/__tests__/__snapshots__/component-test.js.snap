// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`работа Spinner 1`] = `
<div
  className="backdrop"
>
  <div
    className="loader"
  >
    <div
      className="dot"
    />
    <div
      className="dot"
    />
    <div
      className="dot"
    />
    <div
      className="dot"
    />
    <div
      className="dot"
    />
  </div>
</div>
`;
